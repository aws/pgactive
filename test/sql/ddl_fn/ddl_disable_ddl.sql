SELECT pgactive.pgactive_replicate_ddl_command($DDL$ ALTER DATABASE regression RESET pgactive.skip_ddl_replication; $DDL$);
SELECT pgactive.pgactive_replicate_ddl_command($DDL$ ALTER DATABASE postgres RESET pgactive.skip_ddl_replication; $DDL$);
