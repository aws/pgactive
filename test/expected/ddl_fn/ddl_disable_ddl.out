SELECT pgactive.pgactive_replicate_ddl_command($DDL$ ALTER DATABASE regression RESET pgactive.skip_ddl_replication; $DDL$);
 pgactive_replicate_ddl_command 
---------------------------
 
(1 row)

SELECT pgactive.pgactive_replicate_ddl_command($DDL$ ALTER DATABASE postgres RESET pgactive.skip_ddl_replication; $DDL$);
 pgactive_replicate_ddl_command 
---------------------------
 
(1 row)

